### PR TITLE
Change "SHOULD NEVER" to "MUST NOT"

### PR DESCRIPTION
**Pull request by ianbjacobs**
“Should never” is strong enough that if that is what we mean, then
we should use the corresponding RFC 2119 language.

If we mean less than MUST NOT, then let’s say “SHOULD NOT.”

In this edit I swap in MUST NOT for SHOULD NEVER because I think the
intention is the same.

### CODE REVIEW

File: index.html
Comment (marcoscaceres) [inline index.html]: NIT: MUST NOT

**Review by marcoscaceres — CHANGES_REQUESTED** [review]

**Comment by marcoscaceres** [comment]
Whoa,  the commit history on this PR is a bit screwy. 

🔥🚨🔥🚨🔥🚨🔥🚨
Whomever merges this, please make sure you  "squash and merge" and remove the garbage details in the commit history. 
🔥🚨🔥🚨🔥🚨🔥🚨

**Comment by sideshowbarker** [comment]
> Whomever merges this, please make sure you "squash and merge" and remove the garbage details in the commit history.

I think it might need more than just "squash and merge". Not sure but I think some other changes in this branch have not been merged yet; for example https://github.com/w3c/browser-payment-api/pull/356/commits/385bf7d5d41c9326be0374113ddc8683cfaeeb2b.

So I think what should be done before this PR branch is merged is:

1. Strip all commits except https://github.com/w3c/browser-payment-api/pull/356/commits/f86f545c94cc36a76f5fd92f270fd67ba5600da4 (e.g., with a `git rebase -i gh-pages` and then annotating all the other commits with `d` to make git rewrite the branch history with those removed)
2. `git push -f  git@github.com:ianbjacobs/browser-payment-api.git gh-pages` to update the PR

**Comment by ianbjacobs** [comment]
@marcoscaceres, can we drop this pull request and could you simply make the requested change?

**Comment by sideshowbarker** [comment]
@ianbjacobs I have a change ready to push to this PR, but when I tried to push it back to your branch, I got a “permission denied” error.

Can you go to the bottom right-hand side of https://github.com/w3c/browser-payment-api/pull/356 in the GitHub Web UI and make sure the “Allow edits from maintainers” option is checked?

**Comment by ianbjacobs** [comment]
@sideshowbarker, now checked

**Comment by sideshowbarker** [comment]
> @sideshowbarker, now checked

Thanks—OK, I just now force-pushed to update the PR, so @marcoscaceres can merge this

File: index.html
Comment (sideshowbarker) [reply index.html]: > NIT: MUST NOT

Fixed

**Review by marcoscaceres — APPROVED** [review]

**Pull request merged**
